### PR TITLE
Kpmp 2707 rolling bootes into explorer

### PR DESCRIPTION
**Pull request by abarkellumich**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "node-sass-chokidar": "1.5.0",
     "npm-run-all": "4.1.5",
     "papaparse": "5.3.0",
+    "plotly.js": "^1.58.4",
```
Comment (rlreamy) [inline package.json:27]: I thought we were using our kpmp-custom-plotly

File: src/App.js [hunk 2]
```diff
@@ -63,24 +64,25 @@ class App extends Component {
...
+        <ApolloProvider client={apolloClient}>
+          <BrowserRouter basename={baseURL} history={history}>
+            <ErrorBoundaryContainer>
+              <NavBar />
+              <Switch>
+                <Route exact path='/' component={Home} store={store} />
+                <Route exact path='/explorer' component={Explorer} store={store} />
+                <Route exact path='/explorer/genesummary' component={GeneSummaryContainer} store={store} />
+                <Route exact path='/explorer/celltypesummary' component={CellTypeSummaryContainer} store={store} />
+                <Route path='/explorer/dataViz' component={RNASeqVizContainer} store={store} />
```
Comment (rlreamy) [inline src/App.js:76]: This was true before, but some of these routes are exact, and some are not...we should discuss with @zwright 
 whether they can all be exact or not. I think they can.

File: src/initialState.json [hunk 3]
```diff
@@ -1,5 +1,115 @@
 {
   "selectedConcept": {},
   "dataType": "",
-  "tissueType": ""
+  "tissueType": "",
+  "summary": [
```
Comment (rlreamy) [inline src/initialState.json:5]: Do we need all of this in initial state still? Or can some of this be cleaned out?

**Review by rlreamy — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (abarkellumich) [reply package.json:27]: Forgot about the custom module, yes, we should use that instead of this, will update

File: src/App.js (hunk 2 above)
Comment (abarkellumich) [reply src/App.js:76]: FWIW, I was able to, locally, navigate to each route using the in-app routing, and through navigating directly via the URL

File: src/initialState.json (hunk 3 above)
Comment (abarkellumich) [reply src/initialState.json:5]: We will need some of it, in particular the data counts for the summary data, until this turns into real data fetches. But yeah, I think the removal of the dataTypeByParticipant is all worth tossing. That data was being used in a graph that is no longer present

**Review by rlreamy — APPROVED** [review]

**Pull request merged**